shared/src/main/scala/Dictionary.scala

```diff
@@ -0,0 +1,10 @@
+package vyxal
+object Dictionary:
+  var shortDictionary: Seq[String] = Seq()
+  var longDictionary: Seq[String] = Seq()
+  var initalised: Boolean = false
```

Comment:
```suggestion
  private var shortDict: Seq[String] = Seq()
  private var longDict: Seq[String] = Seq()
  private var initialised: Boolean = false

  def shortDictionary: Seq[String] = shortDict
  def longDictionary: Seq[String] = longDict
```

Reply:
Wait actually don't accept this suggestion directly, because the `initialise` method would need to be changed to use `shortDict` and `longDict`

Reply:
Are we really that afraid of overwriting shortDict and longDict?

Reply:
I am afraid of everything